flow-typed/react-flip-move_v2.9.x.js

```diff
@@ -1,25 +1,23 @@
-import type { ComponentType, Element, Node } from 'react';
-
-declare module 'react-flip-move' {
+declare module "react-flip-move" {
   declare export type Styles = {
-    [key: string]: string,
+    [key: string]: string
```

Comment:
Guessing trailing commas aren't kosher in libdefs?

Reply:
Don't think so